fdmi/plugins/sched.c

```diff
@@ -1,31 +1,48 @@
 /* -*- C -*- */
```

Comment:
scheduler can be confusing for general purpose fdmi sample application as its specific to multisite use case. Can we make it generic sample app and change the name to generic name so it can just act as simple FDMI application for reference.

So, m0sched is sample to receive put KV events. May be name it  m0fdmiapp_kv for processing KV events and in future we may have m0fdmiapp_io for object events? Or any other better name we can think of.

Also describe in comments at top of file on what the app does, something like app starts as motr client, registers for fdmi events and processes put kv events and displays the received records. You can elaborate more.

@max-seagate @huanghua78  Any other suggestions?

Reply:
agree

Reply:
m0fdmiapp_io looks too cryptic, this is all I can say for now.

Reply:
other option are  
`fdmi/st/kvsfdmiapp` and `fdmi/st/iofdmiapp` 
or
`fdmi/st/kvs/fdmiapp` and `fdmi/st/io/fdmiapp`

Reply:
I like fdmiapp the only thing is I would not use st as a place for it.

Reply:
Yes this is fine. Two variants are just to write separate samples for KVS events and IO events so it will be simple to read and follow for new developers.

Reply:
What about python part is called fdmi_app and the c-program is fdmi_sample_plugin. The app uses the plugin to deliver its functionality.